lib/storefront/award-biginnovation.svg

```diff
@@ -1,12 +1,101 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 19.2.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!-- Generator: Adobe Illustrator 19.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
https://jakearchibald.github.io/svgomg/ is great, unless there's a minification process in SPA